sys/vtimer/vtimer.c

```diff
@@ -32,9 +34,16 @@
 
 #include "vtimer.h"
 
-#define ENABLE_DEBUG    (0)
```

Comment:
@gebart why did you move this macro upwards? In most scripts you have this line and the `include "debug.h"` one below the other.